sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -2907,6 +2907,15 @@ class DataFrameSuite extends QueryTest
       }
     }
   }
+
+  test("isLocal should consider CommandResult and LocalRelation") {
+    withTable("t1") {
+      val df = sql("CREATE TABLE t USING PARQUET AS SELECT 1 as a")
```

Comment:
can we use `SHOW TABLES` which returns data? Then we don't need to create a table and wrap with `withTable`.

Reply:
OK